src/makeRoutes.js

```diff
@@ -3,19 +3,21 @@ import typeRouter from './routes/notification/type';
 import templateRouter from './routes/notification/template';
 import userAuthRouter from './routes/user/auth';
 import typeAuthRouter from './routes/notification/type/auth';
-import fulfilmentsAuthRouter from './routes/fulfilments/auth';
+import fulfilmentRouter from './routes/fulfilment';
+import fulfilmentAuthRouter from './routes/fulfilment/auth';
 
 const makeRoutes = app => {
   // public routes
   app.use('/public/v1/users', userRouter);
   app.use('/public/v1/notifications/types', typeRouter);
   app.use('/public/v1/notifications/templates', templateRouter);
-  
+  app.use('/public/v1//fulfillment', fulfilmentRouter);
+
 
   // authenticated routes
   app.use('/auth/v1/users', userAuthRouter);
   app.use('/auth/v1/notifications/types', typeAuthRouter);
-  app.post('/auth/v1/fulfilments', fulfilmentsAuthRouter);
+  app.post('/auth/v1/fulfilment', fulfilmentAuthRouter);
```

Comment:
this should be pluralized - fulfillments

Reply:
done